src/app/components/show-players/show-players.view.html

```diff
@@ -0,0 +1,19 @@
+<div class="adt-card"
+        ng-repeat="player in $ctrl.savedPlayers.getAll() | orderBy: 'name' as ordered_players track by player.id"
+        ng-class="{'is-adt-card-selected': $ctrl.isPlayerSelected(player)}"
+        ng-click="$ctrl.togglePlayer(player)">
+    <div class="glyphicon glyphicon-user"></div>
+    <div class="adt-ellipsis-inline" ng-bind="player.name | lowercase"></div>
+    <div class="glyphicon glyphicon-ok adt-card-check"></div>
+</div>
+
+<div class="adt-card">
+    <!-- TODO this string needs to be translated -->
+    <form name="createForm" ng-submit="$ctrl.addNewPlayer()">
+    <button type="submit" class="glyphicon glyphicon-plus adt-button"></button>
+    <input type="text" class="text-center"
+        name="namePlayer"
+        placeholder="add player"
+        ng-model="$ctrl.newPlayerName">
+    </form>
+</div>
```

Comment:
Ogni riga deve finire con un `\n` per considerarsi tale. Ti allego questa domanda su stackoverflow che puo essere un'interessante lettura.
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline